### PR TITLE
fix: align chain id

### PR DESCRIPTION
**Pull request by cgilbe27**
Aligns chain IDs

## Summary by CodeRabbit

- **New Features**
  - Updated network endpoints to reflect new testnet configurations.

- **Documentation**
  - Corrected file path references in documentation for clarity.

- **Tests**
  - Modified test cases to align with the new testnet environment.
  - Updated test suite to use the revised network identifiers.

- **Refactor**
  - Changed internal chain identifiers to maintain consistency with the new testnet naming convention.

### CODE REVIEW

**Comment by Unique-Divine** [comment]
my PR fixes this already

**Comment by Unique-Divine** [comment]
but we'll need to publish for people to use it

**Comment by cgilbe27** [comment]
> my PR fixes this already

Where?

**Review by Unique-Divine — APPROVED** [review]
LGTM, just needs tests passing

File: packages/nibijs/README.md
Comment (Unique-Divine) [inline packages/nibijs/README.md:1]: Looks like this file change is causing the failure in the GH test workflow. You could undo it with:
```
git checkout develop -- pacakges/nibijs/README.md
```

Comment (cgilbe27) [reply packages/nibijs/README.md:1]: image

Comment (cgilbe27) [reply packages/nibijs/README.md:1]: packages was spelt wrong

File: packages/nibijs/src/test/chain.test.ts [hunk 1]
```diff
@@ -30,23 +30,29 @@ describe("chain/chain", () => {
...
+    prefix: string,
+    num: number
+  ) => {
+    expect(result.chainId).toEqual(`nibiru-${prefix}-${num}`)
+    expect(result.chainName).toEqual(`nibiru-${prefix}-${num}`)
+    expect(result.endptGrpc).toEqual(`grpc.${prefix}-${num}.nibiru.fi`)
+    expect(result.endptRest).toEqual(`https://lcd.${prefix}-${num}.nibiru.fi`)
+    expect(result.endptTm).toEqual(`https://rpc.${prefix}-${num}.nibiru.fi`)
     expect(result.feeDenom).toEqual(`unibi`)
   }
```
[the remark this answers is not in the document]
Comment (cgilbe27) [reply packages/nibijs/src/test/chain.test.ts:44]: Thanks for the useless comment

File: packages/nibijs/README.md
Comment (cgilbe27) [reply packages/nibijs/README.md:1]: Ended up just removing this file

File: packages/indexer-nibi/src/heart-monitor.test.ts [hunk 2]
```diff
@@ -95,7 +95,7 @@ const checkFields = (objects: any[], fields: any[]) => {
   })
 }
 
-const nibiruUrl = "itn-3"
+const nibiruUrl = "devnet-2"
 
 const heartMonitor = new HeartMonitor(
   `https://hm-graphql.${nibiruUrl}.nibiru.fi/query`,
```
[the remark this answers is not in the document]
Comment (cgilbe27) [reply packages/indexer-nibi/src/heart-monitor.test.ts:101]: It is devnet-2

**Pull request merged**